test/output-tests.js

```diff
@@ -27,7 +27,10 @@ Object.keys(files).forEach((absPath) => {
 
 describe("output", () => {
   function check(data) {
-    const code = compile(data.actual).code;
+    const code = compile(data.actual, {
+      parse: require("../lib/parsers/acorn.js").parse
```

Comment:
I think Acorn is a better default here, since it's the default anyway, and we have more control over how it parses new syntax.